### DISCUSSION TITLE
Source Voltages and Headroom

### DISCUSSION THREAD

**Discussion by TactileDesign** [kriswilk/buzzah#22] [Q&A]
Kris, 
One more question about V source inputs for the MUX and 2305 driver boards.  Since I am hard wiring my connections in my gen 2 design, I have the choice whether to connect the V source pins of the MUX and 2305 boards to the 5V output or the 3 volt output coming from the RP2040.  On one hand, it would seem to give me more headroom if I use the 5V output.  But, both the mux and 2305 say that can only tolerate up to 5.2 volts - that is a bit too close for comfort since USB power sources can vary a bit.  I guess if the 5V coming from RP2040 is regulated it is OK...  But my question is - does running at 5V give more headroom for the driver boards?  I know the driver software intentionally overdrives the LRAs to minimize start up times, so it seems we might benefit in start up and braking times from more headroom if powering at 5V does indeed get us more headroom.  (Note: if we are maxing out our LRA with 2.5Vrms, that is equivalent to 3.5 V peak - and we are doing this with only 3.2 VDC...  Is there a voltage multiplier in the 2305 rendering this whole discussion moot?)

**Comment by kriswilk** [comment]
The mux and drivers should tolerate any power source that's within USB spec. The original LRA specified runs at 1.8 Vrms which is fine running from the 3.3V/600mA regulator. But if you want to run a larger LRA then yes, you'd want to draw power from the USB supply directly... both for the voltage requirement and possibly the maximum current (remember, there are 2 actuators active during each interval). Just be mindful that if running from a PC (or even a sufficiently "smart" USB battery pack), the current may be limited to 500 mA.
